src/components/Common/FormattedDate.svelte

```diff
@@ -1,22 +1,19 @@
 <script>
-	import {dateFromUnix, formatDate, formatDateRelative, isValidDate} from '../../utils/date';
+	import {dateFromUnix, formatDateCustom, formatDateCustomTooltip, isValidDate} from '../../utils/date';
 	import {configStore} from '../../stores/config';
 
 	export let date = new Date();
 	export let prevDate = null;
 	export let noDate = '';
 	export let prevPrefix = '';
-	export let absolute = false;
```

Comment:
This variable is `export` and can be set from outside. It's set `true` for BeatSavior scores. But we can remove it, though we need to remove usage as well.